pghoard/basebackup_delta.py

```diff
@@ -252,46 +252,76 @@ def _submit_files_in_thread(hexdigest):
 
                     raise BackupFailure("Error while uploading backup files")
 
+        uploaded_size = sum(new_submitted_hashes.values())
+        uploaded_count = len(new_submitted_hashes)
+
+        self.metrics.increase("pghoard.delta_backup_total_size", inc_value=uploaded_size, tags={"site": self.site})
+        self.metrics.gauge("pghoard.delta_backup_upload_size", uploaded_size, tags={"site": self.site})
+        self.metrics.increase("pghoard.delta_backup_total_files", inc_value=uploaded_count, tags={"site": self.site})
+        self.metrics.gauge("pghoard.delta_backup_upload_files", uploaded_count, tags={"site": self.site})
```

Comment:
Maybe also add number of bytes not uploaded (i.e. amount of data that remained the same). That seems to be available as percentage which may be more useful but the absolute number could be interested too.